robot.py

```diff
@@ -25,25 +28,30 @@ def createObjects(self) -> None:
         self.field = wpilib.Field2d()
         wpilib.SmartDashboard.putData(self.field)
 
-        if self.isReal():
-            try:
-                from cscore import CameraServer  # type: ignore
-            except ImportError:
-                self.logger.exception("Could not import CameraServer")
-            else:
-                CameraServer.startAutomaticCapture()
+        # if self.isReal():
+        #     try:
+        #         from cscore import CameraServer  # type: ignore
+        #     except ImportError:
+        #         self.logger.exception("Could not import CameraServer")
+        #     else:
+        #         CameraServer.startAutomaticCapture()
 
     def teleopPeriodic(self) -> None:
-        auto_driving = self.gamepad.getAButton()
+        drop_off = self.gamepad.getAButton()
+        pick_up = self.gamepad.getXButton()
         spin_rate = 6
         drive_x = -rescale_js(self.gamepad.getLeftY(), 0.1) * Chassis.max_wheel_speed
         drive_y = -rescale_js(self.gamepad.getLeftX(), 0.1) * Chassis.max_wheel_speed
         drive_z = -rescale_js(self.gamepad.getRightX(), 0.1, exponential=2) * spin_rate
         local_driving = self.gamepad.getBButton()
 
         self.movement.set_input(vx=drive_x, vy=drive_y, vz=drive_z, local=local_driving)
-        if auto_driving:
-            self.movement.do_autodrive()
+        if drop_off:
+            self.movement.do_autodrive(Pose2d(3, 0, 0), Rotation2d(0))
+        elif pick_up:
+            self.movement.do_autodrive(Pose2d(0, -1, math.pi), Rotation2d(math.pi))
+        else:
+            self.movement.next_state("manualdrive")
```

Comment:
you don't need to explicitly go to the manual drive state as it is the 'default state' and will run if nothing else is running